docs/DeveloperGuide.adoc

```diff
@@ -316,39 +319,141 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Create meme
+
+*MSS*
+
+1. User requests to list meme templates
+2. Weme shows a list of meme template
+3. User selects a template to create a meme
+4. Weme creates the meme from the selected template based on user input
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 2a. The template list is empty
+[none]
+** 2a1. User adds template
++
+2a2. Weme interactively generates template based on user input
++
+Use case ends
+
+* 3a. The given index is invalid
++
+[none]
+** 3a1. Weme shows an error message
++
+Use case resumes at step 2
+
+* 4a. The given input is invalid
```

Comment:
This point was meant for template creation right? I think we should put it in the 'Create template' section

Reply:
Can we call them "Create meme from template" and "Add template"? Otherwise a bit confusing